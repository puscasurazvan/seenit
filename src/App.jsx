import React from 'react'
import EmbeddedVideo from './components/EmbeddedVideo'
import Heading from './components/Heading'
import './App.css'

const videoData = {
  title: 'Seenit Camera Confidence Video',
  src: 'https://youtube.com/embed/OMMY3I1PjoI',
}

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <Heading headingText="How to overcome camera shyness" />
      </header>
      <EmbeddedVideo video={videoData} />
    </div>
  )
}

export default App
