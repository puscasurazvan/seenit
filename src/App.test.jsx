import React from 'react'
import { mount } from 'enzyme'
import App from './App'

describe('App', () => {
  it('Matches the snapshot', () => {
    const wrapper = mount(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
