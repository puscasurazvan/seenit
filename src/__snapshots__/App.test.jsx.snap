// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Matches the snapshot 1`] = `ReactWrapper {}`;
